### DISCUSSION TITLE
Feature request: show created, assigned, mentioned and review requests on the same page

### DISCUSSION THREAD

**Discussion by ricobl** [community/community#21751] [New to GitHub]
<p>I would like to see all pull requests in the same view.</p>
<p>Looks like I can’t tweak the search to include all created, assigned, mentioned and review requests.</p>
<p>Is it possible?</p>

**Comment by majormoses** [comment]
<p>After playing around it does not seem possible as it does not support an <code>or</code> operator and uses an implicit <code>and</code> operator. I found <a href="https://github.com/isaacs/github/issues/660" rel="nofollow noopener">https://github.com/isaacs/github/issues/660</a> which has been asking for the ability for quite a while</p>

**Comment by BelHFL** [comment]
<p>Hi <a class="mention" href="/ricobl">@ricobl</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>

**Comment by ricobl** [comment, marked as answer]
<p>I’ve been playing with the <a href="https://github.com/github/hub" rel="nofollow noopener">hub client</a> and the GraphQL search API.</p>
<p>Also found out about the involves operator that includes most occurrences except requested reviews.</p>
<p>This query gets me what I need:</p>
<pre><code>fragment PullRequest on PullRequest {
  number
  title
  permalink
  author { login }
}

query {
  involves: search(query: "is:open is:pr involves:USERNAME", type: ISSUE, first: 100) {
    nodes { ...PullRequest }
  }
  requests: search(query: "is:open is:pr review-requested:USERNAME", type: ISSUE, first: 100) {
    nodes { ...PullRequest }
  }
}
</code></pre>
<p>That can be run like this:</p>
<pre><code>hub api graphql -F query=@my-pull.graphql
</code></pre>
<p>Leaving this here in case it helps more people.</p>
<p>So instead of using the page I might just use my terminal.</p>

**Comment by mpboom** [comment]
<p>Thanks for posting back the solution!</p>
